docs/docusaurus/docs/guides/connecting_to_your_data/connect_to_data_overview.md

```diff
@@ -28,201 +28,46 @@ Connecting to your data is built around the <TechnicalTag tag="datasource" text=
-
-context = gx.get_context()
-context.test_yaml_config(datasource_yaml)
-```
-
-</TabItem>
-
-<TabItem value="python">
-
-```python
-import great_expectations as gx
-from great_expectations.core.yaml_handler import YAMLHandler
-
-datasource_config = {}  # Replace this with the Python dictionary you want to check for errors.
-
-yaml = YAMLHandler()
-context = gx.get_context()
-context.test_yaml_config(yaml.dump(datasource_config))
-```
-
-</TabItem>
-
-</Tabs>
-
-From here, iterate by editing your config to add config blocks for additional introspection,Data Assets, sampling, etc. After each addition re-run `test_yaml_config()` to verify the addition is functional, then move on to the next iteration of editing your config.
-
-### 4. Save the Datasource configuration to your Data Context.
-
-What is the point of configuring a Datasource if you can't easily use it in the future?  At this point you will want to save your Datasource configuration to your Data Context.  This can be done easily by using the `add_datasource()` function, which is conveniently accessible from your Data Context.  
-
-<Tabs
-  groupId="yaml-or-python"
-  defaultValue='yaml'
-  values={[
-  {label: 'YAML', value:'yaml'},
-  {label: 'Python', value:'python'},
-  ]}>
-
-<TabItem value="yaml">
-
-The function `add_datasource()` takes in a series of named arguments corresponding to the keys in your `datasource_yaml` string.  Fortunately, python and the `yaml` module provide a convenient way to unpack yaml strings into named arguements so you don't have to. 
+The `name` key will be the first you need to define.  The `name` key can be anything you want, but it is best to use a descriptive name as you will use this to reference your Datasource in the future.
+The `add_<datasource>` method will take datasource specific arguments used to configure it. 
+For example, the `add_pandas_filesystem` takes a `base_directory` argument in the example above while the 
+`context.sources.add_postgres(name, ...)` method takes a `connection_string` that is used to connect to the database.
 
-First, you will want to import and initialize the YAMLHandler object:
+Calling the `add_<datasource>` method on your context will run configuration checks. 
+For example, it will make sure the `base_directory` exists for the `pandas_filesystem` datasource and the `connection_string` is valid for a sql database.
 
-```python
-from great_expectations.core.yaml_handler import YAMLHandler
+These methods also persist your datasource to the underlying storage. The storage depends on your <TechnicalTag tag="data_context" text="Data Context" />. 
+For a <TechnicalTag tag="file_data_context" text="File Data Context" /> the changes will be persisted to disk; 
+for a <TechnicalTag tag="cloud_data_context" text="Cloud Data Context" /> the changes will be persisted to the cloud;
+for a <TechnicalTag tag="ephemeral_data_context" text="Ephemeral Data Context" /> the data will remain only in memory.
```

Comment:
I think these don't exist which is causing the netlify build error.

Reply:
I will remove them. Thanks!